orders/tests.py

```diff
@@ -0,0 +1,79 @@
+from django.utils import timezone
+from dogowner.models import DogOwner
+from daycare.models import DayCare
+from .models import Order
+from .models import StatusOptions
+from django.core.exceptions import ObjectDoesNotExist
+import pytest
+
+STARTING_DATE = timezone.now()
+FINISHING_DATE = timezone.now()
+TOTAL_PRICE = 100
+STATUS = StatusOptions.Pending
+
+
+@pytest.fixture
+def order():
+    return Order.create(dog_owner_id=DogOwner.objects.get(pk=1),
+                        daycare_id=DayCare.objects.get(pk=1),
```

Comment:
You should use `objects.get(id=...)` instead of `objects.get(pk=...)`, in case the pk will be changed this code won't be affected and also you are looking by id so its better to be explicit

Reply:
You're right again. I'll change it.